samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1562,6 +1562,34 @@
           }
         ]
       }
+    },
+    "StepFunctionsExecutionPolicy": {
+      "Description": "Gives permission to start a Step Functions state machine execution",
+      "Parameters": {
+        "StateMachineName": {
+          "Description":"The name of the state machine to execute."
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "states:StartExecution"
```

Comment:
Are there other permissions we should include here as well? Are you sure this is the only logical action to give to a function to interact with a state machine? `states:StopExecution` seems to make sense, but looking at [the docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awsstepfunctions.html), it doesn't look like it's scoped to a state machine (which seems weird).

I'm not going to block on this since we could add additional permissions later, but I'll also check around with others to see if any other permissions are logical to add to this policy.

Reply:
I see what you mean and yes it seems weird that it's not scoped to a state machine. The reason why I only include `StartExecution` and not `StopExecution` is that I think in most cases one Lambda won't be responsible for both starting and stopping a state machine. 

I'll leave this as it is and as you said more permissions can be added later.